### DISCUSSION TITLE
Price range calculations

### DISCUSSION THREAD

**Discussion by HannesMorre** [inventree/InvenTree#5704] [Q&A]
How is the article price calculated exactly? I am trying to calculate the exact cost of some of my articles, but there are huge differences in the numbers I see on the part page.
The BOM price range is this: 
![bom ll 01](https://github.com/inventree/InvenTree/assets/71696745/a53a3223-8fec-4ea9-be84-37bb292da466)

But when I look at the pricing tab, I see this: 
![image](https://github.com/inventree/InvenTree/assets/71696745/234a916b-cdf5-4ea2-81a7-54cf17bf1a81)

Where does the 127.1253 € come from?

**Comment by SchrodingersGat** [comment]
@HannesMorre check out the [pricing documentation](https://docs.inventree.org/en/stable/part/pricing/?h=pricing) - it should cover most of this

**Comment by HannesMorre** [comment]
The difference was caused by a substitute part. Thanks anyways.
